modzy/_util.py

```diff
@@ -32,3 +34,44 @@ def file_to_bytes(file_like):
 
     with open(path, 'rb') as file:
         return file.read()
+
+
+def file_to_chunks(file_like, chunk_size):
+    logger.debug(f"file_to_chunks({type(file_like)} :: {file_like}, {chunk_size}) :: splitting ")
+    file = None
+    if not hasattr(file_like, 'read'):
+        if hasattr(file_like, '__fspath__'):  # os.PathLike
+            path = file_like.__fspath__()
+        elif isinstance(file_like, pathlib.Path):  # Python 3.4-3.5
+            path = str(file_like)
+        else:
+            path = file_like
+        file = open(path, 'rb')
+    else:
+        file = file_like
+
+    if hasattr(file, 'seekable') and file.seekable():
+        file.seek(0)
+
+    i = 0
+    while True:
```

Comment:
Might not want to have potentially infinite behavior and implement some `raise StopIteration` condition for this iterator